contribute.html

```diff
@@ -154,6 +154,29 @@ <h2 id="contribute-code-or-docs">Contribute code or documentation<a class="paral
 		<p> If you want to propose a larger change to Astropy, there's a procedure for that:  <a href="https://github.com/astropy/astropy-APEs">Astropy Proposals for Enhancement (APEs)</a> (modeled after Python's <a href="http://legacy.python.org/dev/peps/">PEPs</a>).  The sort of changes that APEs are intended for include plans for new sub-packages, wide-ranging code re-organizations, a new procedure needing review by a lot of the Astropy community, or an informational document on some decision for Astropy that you want remembered.  For more background on APEs, check out <a href="https://github.com/astropy/astropy-APEs/blob/master/APE1.rst">APE #1</a> and the <a href="https://github.com/astropy/astropy-APEs/blob/master/README.rst">APE repository's README</a>.  There is also a <a href="https://github.com/astropy/astropy/wiki/APE-Overview">wiki page on Astropy's github repository</a> that has an overview of the existing APEs.</p>
 
 
+	</section>
+
+	<section id="role">
+		<h2 id="formal-project-role">Formal project role<a class="paralink"
+		href="#formal-project-role" title="Permalink to this headline">¶</a></h2>
+
+		<p>
+            A higher level of contribution to the project comes in the form of assuming a formal
+            project role as listed in the <a href="team.html">Astropy Team</a> page.
```

Comment:
```suggestion
            AIf you are interested in a higher level of contribution to the project, you can consider taking on one of the formal
            project roles as listed in the <a href="team.html">Astropy Team</a> page.
```
For a bit more consistency with the earlier sections which are in second-person